enhancements/metering/slim-install.md

```diff
@@ -0,0 +1,210 @@
+---
+title: slim-install
+authors:
+  - "@timflannagan1"
+reviewers:
+  - TBD
+approvers:
+  - @operator-framework/team-metering
+  - @bparees
+creation-date: 2020-10-06
+last-updated: 2020-10-06
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Slim Installation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+# Summary
```

Comment:
```suggestion
## Summary
```

Reply:
noticed this because our enhancement newsletter tool didn't find the summary section :-)